lib/deimos/active_record_producer.rb

```diff
@@ -44,6 +44,7 @@ def send_events(records, force_send: false)
           generate_payload(attrs, record).with_indifferent_access
         end
         self.publish_list(messages, force_send: force_send)
+        self.post_process(records) if self.respond_to?(:post_process)
```

Comment:
To match the pattern we usually use, I'd add a default `post_process` method to `ActiveRecordProducer` which does nothing, rather than using `respond_to?`.

Reply:
@dorner Updated